server/container/util/src/main/java/org/apache/james/util/BodyOffsetInputStream.java

```diff
@@ -0,0 +1,159 @@
+/****************************************************************
```

Comment:
Oups, this **should be moved** (and shared) from/with mailbox.

Copy and paste evil.